logstash-core/src/main/java/org/logstash/ackedqueue/io/MmapPageIO.java

```diff
@@ -30,33 +30,29 @@ public MmapPageIO(int pageNum, int capacity, String dirPath) {
 
     @Override
     public void open(long minSeqNum, int elementCount) throws IOException {
-        mapFile(STRICT_CAPACITY);
```

Comment:
I think this change will create a dead variable in org.logstash.ackedqueue.io.AbstractByteBufferPageIO (I didn't see it used anywhere else)

Reply:
oh - let me verify that, thanks.

Reply:
I don't see which dead variable you refer to??

Reply:
`org.logstash.ackedqueue.io.AbstractByteBufferPageIO#STRICT_CAPACITY` <- I don't see it used anywhere else.

Reply:
oh right - thanks.